docs/docs/configuration/sessions.md

```diff
@@ -13,6 +13,59 @@ At present the available backends are (as passed to `--session-store-type`):
 - [cookie](#cookie-storage) (default)
 - [redis](#redis-storage)
 
+
+### Generating a Cookie Secret
+
+The cookie containing session data is encrypted by a string called a cookie secret.   
```

Comment:
```suggestion
For encrypting the session data stored inside a `cookie` a `cookie secret` needs to be generated.
```

Reply:
I'm not sure about this one. The documentation as-is don't clearly state that the session cookie is encrypted. I prefer this info to be explicit rather than implicit

Reply:
I made a new suggestion for it :)